tools/privacyidea-token-janitor

```diff
@@ -344,7 +356,7 @@ def export_user_data(token_list):
                 user.realm
             )
         else:
-            uid = u"N/A"
+            uid = u"N/A, N/A, N/A, N/A, N/A, N/A, N/A"
```

Comment:
since we do not process this information, do we need to add all the 'N/A's?

Reply:
The thing is, that the last column can contain the number of tokens assigned to this user.
N/A will sum up the tokens with no user,
So we need that much commas(?) so that in an excel sheed the sum will be in one column.
We could however do
"N/A, , , , , , "
What do you think?

Reply:
change this in 63ffbebb

Reply:
well, i don't like excel... :-)